src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/SVAnnotateEngine.java

```diff
@@ -338,12 +344,14 @@ protected static String annotateBreakend(final SimpleInterval variantInterval,
      * Add consequence of structural variant on an overlapping transcript to consequence dictionary for variant
      * @param variantInterval - SimpleInterval representing structural variant
      * @param svType - SV type
+     * @param isComplex - boolean: true if SV type is CPX, false if not
      * @param transcript - protein-coding GTF transcript
      * @param variantConsequenceDict - running map of consequence -> feature name for variant to update
      */
     @VisibleForTesting
     protected void annotateTranscript(final SimpleInterval variantInterval,
                                              final GATKSVVCFConstants.StructuralVariantAnnotationType svType,
+                                             final boolean isComplex,
```

Comment:
I've been thinking about this and I'm not sure of the optimal names here. 
* I agree that the `StructuralVariantAnnotationType` is being overused here, although I agree that it may not be worthwhile to create two different types with and without CPX. Maybe if it was just `StructuralVariantType` for a more neutral category - especially since it is getting used in other tools now?
* I think `variantInterval` is not inaccurate, since it is an interval of an SV / an interval that varies compared to the reference, and it is more descriptive than `segmentInterval` when distinguishing between a feature interval and a variant interval. But it is true that only one segment is considered at a time - which is clear because the `variantInterval` is a single interval rather than a list. Maybe best would be `variantSegmentInterval`? Or some documentation changes?
* `variantConsequenceDict` is the correct name because it accumulates consequences across SV segments
* True, `annotateGeneOverlaps` and `annotateTranscript` are only applied to one segment at a time. But the method names focus on the features from the GTF rather than saying they are applied to a variant vs. a segment. Perhaps just a docs change here?